tests/common/file_utils_test.py

```diff
@@ -0,0 +1,96 @@
+# pylint: disable=no-self-use,invalid-name
+import os
+import pytest
+
+import responses
+
+from allennlp.common.file_utils import url_to_filename, filename_to_url, get_from_cache, cached_path
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestFileUtils(AllenNlpTestCase):
+
+    def test_url_to_filename(self):
+        for url in ['http://allenai.org', 'http://allennlp.org',
+                    'https://www.google.com', 'http://pytorch.org']:
+            filename = url_to_filename(url)
+            assert "http" not in filename
```

Comment:
This is probably good enough, but if you want to _actually_ check that you have something that's a valid filename, you could touch `os.path.join(self.TEST_DIR, filename)`.